### DISCUSSION TITLE
SO 2.3.140 Elastic Compatibility

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#8241] [Unsupported Versions] [answered]
Question? Does this mean that Security Onion will be compatible with Elastic Agent starting in version 2.3.140?

**Comment by TOoSmOotH** [comment, marked as answer]
We are currently focused on making sure the existing components are compatible with 8.3.X. Although the Elastic Agent might work, it will not be officially supported in .140. We will continue to review additional Elastic components and their place in the platform when time allows.

**Reply by ColeVan** [reply]
Sounds good! I look to the new release and testing some new ideas.
